woocommerce/woocommerce-android
# Add toolbar for CardReaderOnboardingFragment

Part of #10570 

### Description

This PR introduces individual toolbars for CardReaderOnboardingFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Payment hub screen(Menu -> Payments -> onboarding screen) screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="240" alt="Screenshot 2024-02-12 at 7 11 39 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/d1b32586-d8f3-4712-8591-3ac26186662b">
